pkgs/development/python-modules/deepdiff/default.nix

```diff
@@ -8,19 +8,20 @@
 , numpy
 , pytestCheckHook
 , pyyaml
+, pythonOlder
 }:
 
 buildPythonPackage rec {
   pname = "deepdiff";
-  version = "5.7.0";
+  version = "5.8.2";
   format = "setuptools";
 
-  # pypi source does not contain all fixtures required for tests
+  disabled = pythonOlder "3.6";
+
   src = fetchFromGitHub {
     owner = "seperman";
     repo = "deepdiff";
-    # 5.7.0 release not tagged https://github.com/seperman/deepdiff/issues/300
-    rev = "f2ffdb83b2993f4f0bb7e854620f0acd0bf6339e";
+    rev = "v${version}";
     hash = "sha256-0UBx7sH2iMrLVl5FtHNTwoecLHi8GbInn75G3FSg4gk=";
   };
 
```

Comment:
No change in source hash!!  Noticed in https://github.com/NixOS/nixpkgs/pull/173364#issuecomment-1131254806